apps/condo/domains/meter/utils/meter.utils.js

```diff
@@ -0,0 +1,73 @@
+const { isString, isNumber, isUndefined,
+    isEmpty, get, pick } = require('lodash')
+
+/**
+ * @param value
+ * @return {undefined|string|null}
+ */
+function normalizeMeterValue (value) {
+    if (!(isString(value) || isNumber(value) || isUndefined(value))) return null
+    if (isUndefined(value)) return value
+
+    const transformedValue = String(value).trim().replaceAll(',', '.')
```

Comment:
What's this used for? If it's import util, move it to them....

Reply:
If it's API util, should not modify user data. Only validate them. Since writing numbers is locale-specific. API must only accept `xyz.xyz` format

Reply:
made another task for that. now marked with @deprecated and TODO